src/python_testing/TC_TIMESYNC_2_1.py

```diff
@@ -29,45 +29,42 @@ async def read_ts_attribute_expect_success(self, endpoint, attribute):
         cluster = Clusters.Objects.TimeSynchronization
         return await self.read_single_attribute_check_success(endpoint=endpoint, cluster=cluster, attribute=attribute)
 
-    def pics_TC_TIMESYNC_2_1(self) -> list[str]:
-        return ["TIMESYNC.S"]
-
     @async_test_body
     async def test_TC_TIMESYNC_2_1(self):
-
         endpoint = self.user_params.get("endpoint", 0)
```

Comment:
Actually, while you're in here, can you remove this endpoint check? The spec got changed after the test, so this cluster is now mandated to be on EP0 and only EP0, so this is just 0 always. ie, here just do endpoint=0

Reply:
Thank you, I have updated endpoint variable to be statically set to 0 in this test module.